### DISCUSSION TITLE
Is there someting similar to no-leaked-conditional-rendering

### DISCUSSION THREAD

**Discussion by dohomi** [biomejs/biome#4170] [Ideas]
Hi Guys,

is inside of Biome something similar to this eslint rule:

no-leaked-conditional-rendering

https://eslint-react.xyz/docs/rules/no-leaked-conditional-rendering

Its helpful especially working with React Native

**Comment by ematipico** [comment]
No, there isn't. Please use https://github.com/biomejs/biome/discussions/3 to propose new rules from other sources
